### DISCUSSION TITLE
Welcome to ibmi-tobi Discussions!

### DISCUSSION THREAD

**Discussion by edmundreinhardt** [IBM/ibmi-tobi#139] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by beckhamrryyaann** [comment]
My name is Ryan.  I am an IBM i developer writing in **free RPGLE, SQL, CLLE, etc and also write in Node.js, kotlin, php in the open source world all of which i interact with the IBM i.  I am currently on a mission to modernize our companies code base written mostly in RPG IV to using Code for IBM i to write code locally on pc, storing code on a gitlab server using BOB to build the projects.   We have paid copies of RDi but I'm finding Code for IBM i works better.  (especially for local development deploying to IFS)  I appreciate all your work on BOB.  I have successfully converted one application from RPG IV stored in source members in QSYS.LIB to the modern approach described above.  I am excited for the future of IBM i development.

**Reply by edmundreinhardt** [reply]
Welcome Ryan.  I really believe that it is people like you that are our hope for the future.

**Comment by angelorpa** [comment]
Hello there, I'm Angelo, a rookie developer in IBM i/AS400 writing programs in rpgle fully free form using VS Code with Code for IBM i extension along 1 year ago, but all the dev team at company I'm working use SEU so I did susgget modernize and adopt new ways work, the team was very receptive to the propouse. I like all this new stuff that are doing here, this is a very exiting oportunity to learn and contribuite at same time. I hope to be helpful.

Thanks for this great job that is been doing here.

P.S. Apologies for my english, my native language is spanish.

**Reply by edmundreinhardt** [reply]
Bienvenido.  Yo hablo español también.
I think you are really helping your team to move into the future!
